options/output.md

```diff
@@ -317,3 +317,14 @@ Configure the search to increase the relevance of items in a given category.
 ```
 
 Configure the search to increase the relevance of items in a given group.
+
+## sitemapBaseUrl
+
+```json
+// typedoc.json
+{
+    "sitemapBaseUrl": "https://example.com/sitemap.xml"
```

Comment:
The base URL shouldn't include the site map name, this will result in the site map including urls like "sitemap.xml/index.html"